index.html

```diff
@@ -5,15 +5,18 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="./styles/home.css">
 </head>
 <body>
     <footer>
-        <img src="" alt="">
-        <div class="">
-            <button>Contact Us</button>
-            <button>About Us</button>
-            <button>Join Us</button>
+        <div class="image-footer">
+          <img src="logo2.jpg" alt="" />
         </div>
-    </footer>
+        <div class="footer-button">
+          <button>Contact Us</button>
```

Comment:
If you want a button to redirect you to another page or section you can use anchor tag but don't use a button for redirection, if you want the shape and style of the button you can style the anchor to look like a button by css but not using a button.